srcs/main.c

```diff
@@ -54,8 +46,8 @@ int	main(int argc, char **argv)
 			return (0);
 		}
 		// validate user input (line)
-		minishell.coconut = line;
-		printf("%s\n", minishell.coconut);
+		minishell.prompt = line;
```

Comment:
no no no, the line is not going to be the prompt. If you wanna store it, better call it instruction? instruction_line? not sure
The prompt is the "> " argument in readline function above btw

Reply:
Oh, wait I got it. let's put that back to coconut so we know it's something we don't need in the future and should get rid of. I misunderstood something stupid in how you code worked haha.